libr/core/graph.c

```diff
@@ -206,14 +206,28 @@ static void normal_RANode_print(const RAGraph *g, const RANode *n, int cur) {
 	if (y < -1)
 		delta_y = R_MIN (n->h - BORDER_HEIGHT - 1, -y - MARGIN_TEXT_Y);
 
-	/* print the title */
-	if (cur) {
-		snprintf (title, sizeof (title)-1,
-				"[%s]", n->title);
+	char* shortcut = sdb_get (g->db, sdb_fmt (2, "agraph.nodes.%s.shortcut", n->title), 0);
+	if (shortcut) {
```

Comment:
This series of nested if can be made simpler by keeping the old
```
if (cur) {
		snprintf (title, sizeof (title)-1, "[%s]", n->title);
} else {
		snprintf (title, sizeof (title)-1, " %s", n->title);
}
```
After this you can do something like:
```
if (shortcut) {
		// append [shortcut] to title
		free (shortcut);
}
```